rust/relay/ebpf-turn-router/src/main.rs

```diff
@@ -94,6 +90,18 @@ pub fn handle_turn(ctx: XdpContext) -> u32 {
             xdp_action::XDP_PASS
         }
 
+        Error::InterfaceIpv4AddressNotConfigured
+        | Error::PacketLoop
+        | Error::NoEntry(_)
```

Comment:
Either the eBPF relays work or they don't -- by passing any amount of TURN channel data messages up the stack we run the risk of effectively DoS'ing this relay for everyone.

Now that the eBPF code paths are (or will be soon) well exercised in CI, I think it's safe to drop these.